Cargo.toml

```diff
@@ -102,8 +102,12 @@ rand_xorshift = "0.3"
 # ZIP 32
 aes = "0.8"
 fpe = "0.6"
+zip32 = "0.0"
 
 [profile.release]
 lto = true
 panic = 'abort'
 codegen-units = 1
+
+[patch.crates-io]
+zip32 = { git = "https://github.com/zcash-hackworks/zip32.git", rev = "a5c72c909cf29658718d2aec8eeaa970231532e9" }
```

Comment:
Once https://github.com/zcash-hackworks/zip32/pull/3 is reviewed and merged, and we have a published version of `zip32`, I'll update this PR to replace the patch.